test/packages/input_groups/manifest.yml

```diff
@@ -30,6 +30,7 @@ vars:
   - name: secret_access_key
     type: text
     title: Secret Access Key
+    secret: true
```

Comment:
Could you add an example of field in the test package `good_v2` and `good_input`?